src/components/AppConfig/Vector.tsx

```diff
@@ -36,92 +38,93 @@ interface GrafanaVectorAPISettings {
 
 interface StoreSettings {
   type?: string;
+  authType?: string;
+  authSettings?: AuthSettings;
   qdrant?: QdrantSettings;
   grafanaVectorAPI?: GrafanaVectorAPISettings;
 }
 
-interface Props<T> {
+export interface Props<T> {
   settings?: T;
   onChange: (settings: T) => void;
 }
 
-export function VectorConfig({ settings, onChange }: Props<VectorSettings>) {
+export function VectorConfig({ settings, secrets, secretsSet, onChange, onChangeSecrets }: {
+  settings?: VectorSettings;
+  secrets: Secrets;
+  secretsSet: SecretsSet;
+  onChange: (settings: VectorSettings) => void;
+  onChangeSecrets: (secrets: Secrets) => void;
+}) {
   return (
     <FieldSet label="Vector Settings">
 
-      <Field label="Enabled" description="Enable vector database powered features.">
-        <Checkbox
+      <InlineField label="Enabled" tooltip="Enable vector database powered features.">
+        <InlineSwitch
```

Comment:
This was intentionally moved to a `Checkbox` rather than an `InlineSwitch` in a [previous commit](2d0d8804) after a review comment from a frontend dev mentioned that it's the recommended UX in Grafana.

In general it's easier to review if these other UI changes aren't scattered throughout the PR 😢